doc/whats_new.rst

```diff
@@ -247,6 +247,12 @@ API changes summary
      needed for the perplexity calculation. :issue:`7954` by
      :user:`Gary Foreman <garyForeman>`.
 
```

Comment:
Hi, I tried to organize into section each entry in API change summary. This resulted the conflict. I resolved it but appveyor test still failed (without merge). Should I remove my last commit ?